### ISSUE TITLE
Pretraining VLM datasets and Training parameters

### ISSUE THREAD

**Issue by OscarHuangWind** [ByteDance-Seed/Bagel#225]
Hi, thank you for open-sourced code!

I am planning to pretrain the Qwen2.5 0.5B/3B with NaViT over the general knowledge datasets (Actually, I found that the VLM part of Bagel cannot load the pre-trained Qwen2.5-VL because of the dimension mismatch, it seems that the VLM part of the Bagel is manually trained, am I right?).

I am wondering what kind of and what size of the datasets should I used for training (is BLIP558K used by LLaVA 1.5 sufficient?), and can you suggest the training parameters as well as estimated training time on 4 H100 gpus?

Many thx!

**Comment by Andy1621** [comment]
Good question! We train the VLM part from pure LLM + ViT. 

I don't suggerst training VLM part with such small datasets. You can try to combine fixed Qwen2.5-VL with diffusion~

**Comment by OscarHuangWind** [comment]
> Good question! We train the VLM part from pure LLM + ViT.
> 
> I don't suggerst training VLM part with such small datasets. You can try to combine fixed Qwen2.5-VL with diffusion~

Thank you for your reply
I am currently employing this MoT architecture into my robot learning project, and therefore, that's why I prefer the small size one.

I wonder is it possible to share with me 0.5B or smaller than 7B size pretrained Bagel model? This would significantly be benefit since we don't need tune the VLM part by ourselves.
